src/server/transaction.cc

```diff
@@ -1112,12 +1123,20 @@ pair<bool, bool> Transaction::ScheduleInShard(EngineShard* shard) {
     if (now >= last_log_time + 10) {
       last_log_time = now;
       EngineShard::TxQueueInfo info = shard->AnalyzeTxQueue();
-      LOG(WARNING) << "TxQueue is too long. Tx count:" << info.tx_total
-                   << ", armed:" << info.tx_armed << ", runnable:" << info.tx_runnable
-                   << ", total locks: " << info.total_locks
-                   << ", contended locks: " << info.contended_locks << "\n"
-                   << "max contention score: " << info.max_contention_score
-                   << ", lock: " << info.max_contention_lock_name;
+      string msg =
+          StrCat("TxQueue is too long. Tx count:", info.tx_total, ", armed:", info.tx_armed,
+                 ", runnable:", info.tx_runnable, ", total locks: ", info.total_locks,
+                 ", contended locks: ", info.contended_locks, "\n");
+      absl::StrAppend(&msg, "max contention score: ", info.max_contention_score,
```

Comment:
Any reason to use `StrAppend()` instead of continuing with the previous statement?

Reply:
for aesthetic reasons :)